internal/datacoord/server.go

```diff
@@ -789,7 +792,7 @@ func (s *Server) GetVChanPositions(channel string, collectionID UniqueID, seekFr
 			continue
 		}
 
-		unflushed = append(unflushed, trimSegmentInfo(s.SegmentInfo))
+		unflushed = append(unflushed, s.SegmentInfo)
```

Comment:
Why change remove trimming logic for `unflushed` segments while keep it on flushed ones? Trimming here is for watch dml channel task too large to store in etcd while performing `LoadCollection`. Is it safe to remove it here?

Reply:
Query node need to load growing segment (unflushed segment here) from storage, so datacoord need to pass binlog path to querycoord and querynode by GetRecoveryInfo interface. I'v been discussed with @sunby, we all think binlog path of `unflushd segment` would not be too large.